content/docs/command-reference/exp/remove.md

```diff
@@ -26,7 +26,8 @@ With `--queue`, the list of experiments awaiting execution is cleared instead.
 - `--queue` - remove all experiments that haven't been run yet (defined via
   `dvc exp run --queue`).
 
-- `-A`, `--all` - remove all experiments (includes `--queue`).
+- `-A`, `--all` - remove all experiments that have been run. Use `--queue` to
+  remove queued ones.
```

Comment:
Does this apply as well for `exp push/list/pull` ? Cc @dberenbaum and @karajan1001 (per #3191-3)

Reply:
And what about `exp show -a/A`? Should we mention that in that case queued experiments ARE included in the output?

Reply:
> Does this apply as well for exp push/list/pull ? Cc @dberenbaum and @karajan1001 (per #3191-3)

Currently `push/list/pull` does not work on queued ones. 

> And what about exp show -a/A? Should we mention that in that case queued experiments ARE included in the output? May be a product question on whether the behavior is consistent, not sure.

This is of course a problem. Cc @pmrowla .

Reply:
Yeah, they are currently excluded from everything except for `exp run --queue`, and `exp remove --queue`, and `exp show`. For now, I think that behavior is fine if correctly documented.

Eventually, queue operations should be in a separate command. See https://github.com/iterative/dvc/pull/7275#issuecomment-1021947145 and https://github.com/iterative/dvc/issues/5615.

Reply:
> And what about exp show -a/A? Should we mention that in that case queued experiments ARE included in the output?

Just wanted to clarify that queued experiments will be shown even without `-a/A` flag.